util.py

```diff
@@ -4,7 +4,15 @@
 import numpy as np
 import shapely
 from glob import glob
-
+from os import listdir
+
+# Lists of available brains and maps
+# On second thoughts, I have no idea why I just put this here.
+# I guess I was thinking it was an Enum despite moving it about
+# purely because it wasn't. Hmmm. Oh well, it's here now.
+# Maybe it'll be useful.
+BRAIN_LIST = sorted([s[:-3] for s in listdir("brains/") if "Brain.py" in s])
```

Comment:
I think this could actually be changed to a function.
```
def class_listing(folder, to_match):
    return sorted([s[:-3] for s in listdir(folder) if to_match in s])
```
But also we could just revert https://github.com/ShefBots/piwarsimulator/pull/2/commits/8627b82e946262a2d6533d303bff85a062b1f4d9

Reply:
See https://github.com/ShefBots/piwarsimulator/pull/2#discussion_r1568903186

On the function point of things, we could wrap it in a function, but for speed (I don't know if the cpython bytecode assembly would optimise it away, but I'd rather be sure), I recommend having the variable as a module member and then the access function just use that, as I've done in for example: https://github.com/ShefBots/omnicam-zero/blob/374eba56b3bc6dc11a554efd119b2a3d58c22f44/protocol.py#L24-L30

Reply:
The `_LIST` stuff is only ever used once for the argument list. I'd say optimisation of it is not a concern.

Reply:
As per https://github.com/ShefBots/piwarsimulator/pull/2#discussion_r1569085401 I'll just move it back out of util because it doesn't need to be in there currently.